### PR TITLE
Fix/modul cabang

### PR DESCRIPTION
**Pull request by dionrespati**

### CODE REVIEW

**Review by aribowobob — CHANGES_REQUESTED** [review]

File: src/modules/system/product/form-product.tsx [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+  open,
+  onSubmit,
+  onCancel,
+}) => {
+  const [formData, setFormData] = useState<IProductProps2>(dataAwal);
+
+  console.log({ formData });
+
+  const [isShowDeleteConfirm, setIsShowDeleteConfirm] = useState(false);
+  const [isEditForm, setIsEditForm] = useState(false);
```
[the remark this answers is not in the document]
Comment (dionrespati) [reply src/modules/system/product/form-product.tsx:32]: FIXED

File: src/modules/system/product/form-product.tsx [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+
+  useEffect(() => {
+    if (id === 0 || id === undefined || id === null) {
```
[the remark this answers is not in the document]
Comment (dionrespati) [reply src/modules/system/product/form-product.tsx:35]: FIXED

File: src/modules/system/product/form-product.tsx [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+            value={nama_satuan}
+            onChange={handleChange}
+            className="mb-4"
+            message="Boleh dikosongkan. Apabila dikosongkan akan diberi nilai default
+          “unit”"
+          />
+          {isEditForm ? (
+            <div className="flex gap-4">
+              <Button
+                className="w-1/4"
```
[the remark this answers is not in the document]
Comment (dionrespati) [reply src/modules/system/product/form-product.tsx:122]: FIXED

File: src/modules/system/product/form-product.tsx [hunk 4]
```diff
@@ -0,0 +1,159 @@
...
+                color="danger"
+                ghost
+                onClick={() => setIsShowDeleteConfirm(true)}
+              >
+                Hapus
+              </Button>
+              <Button className="w-3/4" onClick={() => handleSubmit()}>
```
[the remark this answers is not in the document]
Comment (dionrespati) [reply src/modules/system/product/form-product.tsx:129]: FIXED

File: src/modules/system/product/form-product.tsx [hunk 5]
```diff
@@ -0,0 +1,159 @@
...
+                Simpan Perubahan
+              </Button>
+            </div>
+          ) : (
+            <Button className="w-full" onClick={() => handleSubmit()}>
```
[the remark this answers is not in the document]
Comment (dionrespati) [reply src/modules/system/product/form-product.tsx:134]: FIXED

**Review by aribowobob — APPROVED** [review]

**Pull request merged**